### PR TITLE
modified code to run only stylegan

### PR DESCRIPTION
**Pull request by swamiviv**
Creating a PR so I can comment on things

### CODE REVIEW

File: training/coach.py [hunk 1]
```diff
@@ -113,11 +113,15 @@ def init_models(self):
                                                     mode=self.args.mode_enc, opts=self.args).to(self.device)
 
         # initialize decoder
-        self.decoder = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=2,
+        # self.decoder = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=2,
+        #                          n_mlp=self.args.n_mlp, channel_multiplier=self.args.channel_multiplier).to(self.device)
+        self.decoder = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=0,
```
Comment (swamiviv) [inline training/coach.py]: Pass c_dim as an argument or obtain it from the classifier.

File: training/coach.py [hunk 2]
```diff
@@ -113,11 +113,15 @@ def init_models(self):
...
                                  n_mlp=self.args.n_mlp, channel_multiplier=self.args.channel_multiplier).to(self.device)
 
         # initialize decoder to keep ema
-        self.decoder_ema = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=2,
+        # self.decoder_ema = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=2,
+        #                          n_mlp=self.args.n_mlp, channel_multiplier=self.args.channel_multiplier).to(self.device)
+        self.decoder_ema = Generator(self.args.output_size, style_dim=self.args.latent_dim, c_dim=0,
```
Comment (swamiviv) [inline training/coach.py]: Same as above

File: training/coach.py [hunk 3]
```diff
@@ -331,46 +346,69 @@ def train(self):
 
                 ### encoder part
                 # get encodings
-                encoder_rep_x_2 = self.encoder(x_2)
+                # encoder_rep_x_2 = self.encoder(x_2)
```
Comment (swamiviv) [inline training/coach.py:345]: Line 338: Both real and aug images to generator loss seems to be unaugmented.

File: training/coach.py [hunk 4]
```diff
@@ -331,46 +346,69 @@ def train(self):
...
-                    loss_type=which_loss
-                )
+                # which_loss = ["clf"]
+                # cycle_loss, cycle_loss_dict, _ = self.calc_loss(
+                #     x=x_2,
+                #     y_hat=y_2_hat,
+                #     loss_type=which_loss
+                # )
+                cycle_loss = 0
+                cycle_loss_dict = {}
```
Comment (swamiviv) [inline training/coach.py]: The GAN training does not use any recon loss. Just adversarial losses.
